### ISSUE TITLE
Frozen Protein Foundation-Model Embeddings Improve Antibody–Antigen Binding Affinity Prediction

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#560]
**Authors:** Roger Wang; Kevin Jin; Lurong Pan

**Venue:** bioRxiv
**Date:** 2026-07-14

**Relevance:** 0.950639

## Identifiers

- DOI: [10.64898/2026.07.13.738250](https://doi.org/10.64898/2026.07.13.738250)
- Source: biorxiv `10.64898/2026.07.13.738250`
- Paper: https://www.biorxiv.org/content/10.64898/2026.07.13.738250v1

## Abstract

We investigate whether representations from AINN-P1—a protein foundation model trained autoregressively on tens of millions of natural protein sequences—transfer to the task of ranking antibody–antigen pairs by binding affinity. Casting affinity maturation as a learning-to-rank problem over the change in binding free energy (ΔΔG), we compare a task-specific sequence model trained end-to-end from scratch against lightweight downstream heads built on top of frozen AINN-P1 embeddings, all evaluated under an identical five-fold cross-validation protocol. A regularized linear probe on the frozen embeddings already surpasses the from-scratch baseline, and an optimized lightweight head raises the mean Spearman rank correlation from 0.42 to 0.53—a relative improvement of approximately 28%— while training in seconds and without any fine-tuning of the foundation model. Because a linear probe alone exceeds the fully trained end-to-end baseline, the gain is attributable to representation quality rather than to added downstream-model capacity. These results position frozen foundation-model embeddings as a strong, data-efficient default for affinity ranking in antibody engineering and establish a conservative lower bound that task-adaptive fine-tuning is expected to exceed.

## BibTeX

The collision-checked citation key is `wang2026`; the entry below is ready to copy.

```bibtex
@misc{wang2026,
  title = {Frozen Protein Foundation-Model Embeddings Improve Antibody–Antigen Binding Affinity Prediction},
  author = {Roger Wang and Kevin Jin and Lurong Pan},
  year = {2026},
  abstract = {We investigate whether representations from AINN-P1—a protein foundation model trained autoregressively on tens of millions of natural protein sequences—transfer to the task of ranking antibody–antigen pairs by binding affinity. Casting affinity maturation as a learning-to-rank problem over the change in binding free energy (ΔΔG), we compare a task-specific sequence model trained end-to-end from scratch against lightweight downstream heads built on top of frozen AINN-P1 embeddings, all evaluated under an identical five-fold cross-validation protocol. A regularized linear probe on the frozen embeddings already surpasses the from-scratch baseline, and an optimized lightweight head raises the mean Spearman rank correlation from 0.42 to 0.53—a relative improvement of approximately 28\%— while training in seconds and without any fine-tuning of the foundation model. Because a linear probe alone exceeds the fully trained end-to-end baseline, the gain is attributable to representation quality rather than to added downstream-model capacity. These results position frozen foundation-model embeddings as a strong, data-efficient default for affinity ranking in antibody engineering and establish a conservative lower bound that task-adaptive fine-tuning is expected to exceed.},
  doi = {10.64898/2026.07.13.738250},
  url = {https://doi.org/10.64898/2026.07.13.738250},
  note = {Version 1},
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **Frozen AINN-P1 embeddings improve antibody–antigen ΔΔG ranking over an in-house sequence model trained from scratch** [@wang2026]. Under the same five-fold cross-validation splits, a linear probe on frozen embeddings increased mean Spearman \(\rho\) from 0.417 to 0.457, while an optimized nonlinear head reached 0.533 (Table 1; Figure 2). The controlled folds support a within-dataset comparison, but the evidence is weak-to-moderate because the dataset size and composition, fold-level scores, uncertainty, statistical tests, model architecture, and baseline training procedure are undisclosed; no public PLM, sequence, or structure baseline was tested. This supports the repository’s observations that generic PLM representations can encode antibody-affinity signal, but is narrower than those public benchmarks. Related: [[Generic PLMs outperform antibody-specific PLMs on zero-shot predictions of affinity changes]]; [[Mean-pooled embeddings outperform other zero-shot approaches for transfer learning of PLMs using the full sequence]]; [[ESM-IF outperforms ProteinMPNN on Ab inverse folding and affinity prediction]]. Placement: new atomic note, `Frozen PLM embeddings improve antibody-antigen affinity ranking over training from scratch.md`.

- **Downstream-head nonlinearity contributes more absolute Spearman improvement than replacing the from-scratch representation with frozen AINN-P1 embeddings** [@wang2026]. Table 1 shows a \(0.040\) increase from the from-scratch model to the frozen linear probe, but a \(0.076\) increase from the linear probe to the optimized nonlinear head. This directly contradicts the Discussion’s statement that “the largest single jump” comes from replacing the representation rather than adding downstream capacity; however, causal attribution remains impossible because both the baseline and optimized-head architectures are proprietary (§§4.3–4.4). Related: [[Mean-pooled embeddings outperform other zero-shot approaches for transfer learning of PLMs using the full sequence]]; [[Sparse autoencoder-derived features do not outperform PLM-derived embeddings for downstream prediction]]. Placement: include as a qualification in the Details of `Frozen PLM embeddings improve antibody-antigen affinity ranking over training from scratch.md`, not as a separate note.

- **The reported five-fold cross-validation does not establish generalization to unseen antibody lineages or antigen targets** [@wang2026]. Section 4.5 states only that identical folds were used; it does not report whether homologous antibodies, related antigens, or variants of the same complex were grouped across folds. The dataset is proprietary, and the paper provides neither its sample count nor an external test set (Data and code availability), so training–test similarity cannot be audited despite being especially important for protein and antibody modeling. This substantially limits the scope of the reported \(\rho=0.533\) and qualifies, rather than contradicts, the repository’s affinity-prediction notes. Related: [[Correct antibody-antigen prediction in AF3 and related models is partially determined by training set similarity]]; [[Protein language models are better zero-shot predictors for ranking closely related sequences than distantly related sequences]]; [[Antibody-antigen modeling by diffusion-based structure prediction is data-limited]]. Placement: include as a limitation in `Frozen PLM embeddings improve antibody-antigen affinity ranking over training from scratch.md`, not as its own finding note.

- **The study does not test whether task-adaptive fine-tuning improves AINN-P1 affinity ranking** [@wang2026]. The Discussion calls the frozen result a “conservative lower bound” that fine-tuning is expected to exceed, but Table 1 contains no fine-tuned AINN-P1 configuration and §§4.2–4.3 explicitly keep the encoder frozen. This expectation is therefore speculative and should not be recorded as a finding, particularly because existing evidence in the repository shows that fine-tuning can either help or harm depending on task and data regime. Related: [[Fine-tuning almost always improves property prediction]]; [[Fine-tuning can be detrimental to performance]]; [[Pretraining contributes nearly nothing to performance when fine-tuning protein language models in data-rich situations]]. Placement: include as a limitation in `Frozen PLM embeddings improve antibody-antigen affinity ranking over training from scratch.md`, not as a separate note.

#### Metadata check

Issue #560 contains a complete `@misc` BibTeX entry with citekey `wang2026` and DOI `10.64898/2026.07.13.738250`. Neither the citekey, DOI, nor title currently occurs in `bibliography.bib`. The inspected source is the complete five-page bioRxiv version 1 posted July 14, 2026. No supplementary file is listed; the paper states that the dataset, AINN-P1 model, and architectural and training details are proprietary, leaving only the aggregate values in Table 1 available for verification.
